rpcs3/rpcs3qt/debugger_list.h

```diff
@@ -16,24 +16,28 @@ class debugger_list : public QListWidget
 {
 	Q_OBJECT
 
+	u32 m_start_addr = 0;
+
 public:
 	u32 m_pc = 0;
 	u32 m_item_count = 30;
 	bool m_follow_thread = true; // If true, follow the selected thread to wherever it goes in code
-	u32 m_selected_instruction = -1;
 	QColor m_color_bp;
 	QColor m_color_pc;
 	QColor m_text_color_bp;
 	QColor m_text_color_pc;
 
+	u32 selected_instruction = -1;
+	bool showing_selected_instruction = false;
```

Comment:
```suggestion
	u32 m_selected_instruction = -1;
	bool m_showing_selected_instruction = false;
```

Reply:
These are public members though.

Reply:
it's not about public usage.
It's about readability and code comprehension